service/src/main/java/com/codeforcommunity/processor/TeamsProcessorImpl.java

```diff
@@ -168,4 +182,72 @@ public void inviteToTeam(JWTData userData, InviteMembersRequest inviteMembersReq
       emailer.sendInviteEmail(email, "Team Member", inviter, inviterTeam);
     });
   }
+
+  @Override
+  public GetAllTeamsResponse getAllTeams() {
+    Result<?> teamResult = db.select(TEAM.ID, TEAM.NAME, DSL.count().as("memberCount"))
```

Comment:
@connernilsen You did a lot of these SQL builders, can this be cleaned up at all?

Reply:
I know it's already been merged in, but I think this result can be generic typed to `Result<Record3<Integer, String, Integer>>`, and instead of doing a stream, it would be a lot cleaner to use `fetchInto(TeamSummary.class).

Reply:
I just made those changes on another PR, so don't worry about updating that.